pkg/api/kubtest/model_execution_status.go

```diff
@@ -0,0 +1,20 @@
+/*
+ * Kubtest API
+ *
+ * Kubtest provides a Kubernetes-native framework for test definition, execution and results
+ *
+ * API version: 1.0.0
+ * Contact: kubtest@kubshop.io
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+package kubtest
+
+type ExecutionStatus string
+
+// List of ExecutionStatus
+const (
+	QUEUED_ExecutionStatus  ExecutionStatus = "queued"
```

Comment:
Can we add some kind of unknown status in case of unexpected errors ? I've hit such today :)

Reply:
i'd rather not - it's not very helpful for the user of the API...